pkg/operation/botanist/operatingsystemconfig.go

```diff
@@ -113,6 +125,29 @@ func (b *Botanist) getOperatingSystemConfigCABundle() *string {
 	return &caBundle
 }
 
+func (b *Botanist) getOperatingSystemConfigLokiKubeRBACProxyAuthToken(ctx context.Context) (string, error) {
+	s := b.Secrets[common.StaticTokenSecretName]
+	if s == nil {
+		return "", fmt.Errorf("Missing %s secret", common.StaticTokenSecretName)
+	}
+	data := s.Data[secrets.DataKeyStaticTokenCSV]
+	if len(data) <= 0 {
+		return "", fmt.Errorf("Missing %s in %s secret", common.StaticTokenSecretName, secrets.DataKeyStaticTokenCSV)
+	}
+	tokenRows := strings.Split(string(data), "\n")
+	for _, tokenRow := range tokenRows {
+		splitTokenRow := strings.Split(tokenRow, ",")
+		if len(splitTokenRow) < 3 {
+			return "", fmt.Errorf("Invalid token %v", splitTokenRow)
+		}
+		if splitTokenRow[1] == logging.PromtailRBACName && splitTokenRow[2] == logging.PromtailRBACName {
+			return splitTokenRow[0], nil
+		}
+	}
+
+	return "", fmt.Errorf("Missing promtail auth token")
+}
```

Comment:
Maybe you can do something similar to https://github.com/gardener/gardener/blob/d9f3296f399cafd5ef9a3447d7abae967628f59f/pkg/operation/botanist/secrets.go#L257-L265 ...

Reply:
ok